modules/local/longranger/align/main.nf

```diff
@@ -5,9 +5,9 @@ process LONGRANGER_ALIGN {
     tag "$meta.id"
     label 'process_high'
 
-    def version = '2.2.2-c1'
+    def version = '2.2.2-c2'
 
-    container "gitlab-registry.internal.sanger.ac.uk/tol-it/software/docker-images/longranger:2.2.2-c1"
+    container "gitlab-registry.internal.sanger.ac.uk/tol-it/software/docker-images/longranger:2.2.2-c2"
```

Comment:
Can you use the same version of docker image as the one in GitHub?
`gitlab-registry.internal.sanger.ac.uk/tol-it/software/docker-images/longranger:2.2.2-c4`